tests/console/sshd.pm

```diff
@@ -156,6 +158,8 @@ sub run {
     record_info("Restart sshd", "Restart sshd.service");
     systemctl("restart sshd");
 
+    systemctl('start ' . $self->firewall) if $reenable_firewall;
```

Comment:
Wouldn't it be better to re-enable the firewall in the `post_fail_hook` and `post_run_hook`, so that it will be re-enabled, also if the ssh test fails?

Reply:
Hm, apparently this has `no_rollback` set in public cloud scenarios for some reason, so this would indeed make a difference.

What's the best way to communicate `$reenable_firewall`?

Reply:
I've done something similar using a local test variable in [tests/containers/docker.pm](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/containers/docker.pm#L42). Perhaps that might be helpful for you here.

Reply:
Ok, I just copied that now